packages/catalog-readmodel-writer/test/integration.test.ts

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable functional/no-let */
+/* eslint-disable functional/immutable-data */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import { afterEach, beforeAll, describe, expect, it } from "vitest";
+import {
+  EServiceCollection,
+  ReadModelRepository,
+  consumerConfig,
+} from "pagopa-interop-commons";
+import {
+  EServiceAddedV1,
+  EServiceEvent,
+  EServiceEventEnvelope,
+  EServiceTechnologyV1,
+  EventEnvelope,
+} from "pagopa-interop-models";
+import { v4 as uuidv4 } from "uuid";
+import { GenericContainer } from "testcontainers";
+import { handleMessage } from "../src/consumerService.js";
+
+describe("database test", async () => {
+  let eservices: EServiceCollection;
+
+  const config = consumerConfig();
+  beforeAll(async () => {
+    const mongodbContainer = await new GenericContainer("mongo:4.0.0")
+      .withEnvironment({
+        MONGO_INITDB_DATABASE: config.readModelDbName,
+        MONGO_INITDB_ROOT_USERNAME: config.readModelDbUsername,
+        MONGO_INITDB_ROOT_PASSWORD: config.readModelDbPassword,
+      })
+      .withExposedPorts(27017)
+      .start();
+
+    config.readModelDbPort = mongodbContainer.getMappedPort(27017);
+
+    const readModelRepository = ReadModelRepository.init(config);
+    eservices = readModelRepository.eservices;
+  });
+
+  afterEach(async () => {
+    await eservices.deleteMany({});
+  });
+
+  describe("Handle message for eservice creation", () => {
+    it("should create an eService", async () => {
+      const id = uuidv4();
```

Comment:
Same, should we use `generateId()` instead?

Reply:
Sure